cluster-provision/gocli/cmd/provision.go

```diff
@@ -51,6 +60,36 @@ func provision(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	version, err := cmd.Flags().GetString("k8s-version")
+	if err != nil {
+		return err
+	}
+
+	if len(args) == 1 {
+		packagePath := args[0]
+		if version == "" {
+			versionBytes, err := ioutil.ReadFile(filepath.Join(packagePath, "version"))
+			if err != nil {
+				return err
+			}
+			version = strings.TrimSpace(string(versionBytes))
+		}
+		baseBytes, err := ioutil.ReadFile(filepath.Join(packagePath, "base"))
+		if err != nil {
+			return err
+		}
+		base = fmt.Sprintf("kubevirtci/%s", strings.TrimSpace(string(baseBytes)))
+
+		prefix = fmt.Sprintf("k8s-%s-provision", filepath.Base(packagePath))
+		target = fmt.Sprintf("kubevirtci/k8s-%s", filepath.Base(packagePath))
+		if scripts == "" {
+			scripts = filepath.Join(packagePath)
+		}
+	} else {
+		base = args[0]
+		target = args[1]
+	}
```

Comment:
We use gocli provision with one argument at `publish.sh` like `../gocli/build/cli provision ${provision_dir}` maybe we can remove this.

Reply:
Done.